javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,39 @@
+// Housey pricey (A house price estimator)
+
+/* Two of your friends are considering buying a house, but cannot figure out what the right price should be. 
+Your friends know the width, the height and the depth of the house and the garden size. 
+Lets help them figure out if they paid too much */
+
+class House {
+    constructor(width, height, depth, gardenSizeInM2, houseCost) {
+        this.width = width;
+        this.height = height;
+        this.depth = depth;
+        this.gardenSizeInM2 = gardenSizeInM2;
+        this.houseCost = houseCost;
+    }
+    get price() {
+        return this.priceCalculation();
+    }
+    priceCalculation() {
+        const volumeInMeters = this.width * this.height * this.depth;
+        return volumeInMeters * 2.5 * 1000 + this.gardenSizeInM2 * 300;
```

Comment:
Here you use 2.5, 1000 and 300. Replace with variables. For ex. I save 300 in 'const any = 300'
Set default values for these variable in constructor and it will also give the option to the users to pass other value if they want.

Reply:
Thanks for clarifying it. Did I fix it correctly?

Reply:
Try this
constructor(width, height, depth, gardenSizeInM2, houseCost, x=2.5, y=1000, z=300) {
        this.width = width;
        this.height = height;
        this.depth = depth;
        this.gardenSizeInM2 = gardenSizeInM2;
        this.houseCost = houseCost;
        this.x = x;
        this.y = y;
        this.z = z;
    }